### ISSUE TITLE
Force Exact Match for Observed Keyframes

### ISSUE THREAD

**Issue by Neleac** [setarehc/diffusion-motion-inbetweening#26]
I'm building a tool that requires the observed keyframes to be unchanged. What's the best way to do this? 

I trained the model with `zero_keyframe_loss = True` and velocity loss to mitigate jitter. But there is always a pop between the generated and ground truth at the observed keyframes.
